site/content/docs/4.3/components/dropdowns.md

```diff
@@ -560,12 +560,12 @@ You can also create non-interactive dropdown items with `.dropdown-item-text`. F
 
 ### Active
 
-Add `.active` to items in the dropdown to **style them as active**.
+Add `.active` to items in the dropdown to **style them as active**. Regarding accessibility, the active state should also be indicated with the `aria-current` attribute — using the `page` value for current page, or `true` for current item in a set.
```

Comment:
Sounds a bit stilted. Suggest something more like

"Regarding accessibility, the active state should also be indicated with" > "To convey the active state to assistive technologies, use"

"for current" > "for the current"